apiserver/facades/controller/crossmodelrelations/register.go

```diff
@@ -18,21 +19,27 @@ import (
 // Register is called to expose a package of facades onto a given registry.
 func Register(registry facade.FacadeRegistry) {
 	registry.MustRegister("CrossModelRelations", 3, func(stdCtx context.Context, ctx facade.ModelContext) (facade.Facade, error) {
-		return newStateCrossModelRelationsAPI(ctx) // Removes remote spaces
+		return makeStateCrossModelRelationsAPI(stdCtx, ctx) // Removes remote spaces
 	}, reflect.TypeOf((*CrossModelRelationsAPIv3)(nil)))
 }
 
-// newStateCrossModelRelationsAPI creates a new server-side CrossModelRelations API facade
+// makeStateCrossModelRelationsAPI creates a new server-side CrossModelRelations API facade
 // backed by global state.
-func newStateCrossModelRelationsAPI(ctx facade.ModelContext) (*CrossModelRelationsAPIv3, error) {
+func makeStateCrossModelRelationsAPI(stdCtx context.Context, ctx facade.ModelContext) (*CrossModelRelationsAPIv3, error) {
 	authCtxt := ctx.Resources().Get("offerAccessAuthContext").(*common.ValueResource).Value
 	st := ctx.State()
 	m, err := st.Model()
 	if err != nil {
 		return nil, err
 	}
 
+	modelInfo, err := ctx.ServiceFactory().ModelInfo().GetModelInfo(stdCtx)
+	if err != nil {
+		return nil, fmt.Errorf("retrieving model info: %w", err)
```

Comment:
Should we include the the facade we are trying to create here?

Reply:
shouldn't that information be added by the caller?